benchopt/cli/main.py

```diff
@@ -54,6 +54,10 @@
               " are included. Note that <dataset_name> can be a regexp. "
               "To include multiple datasets, use multiple `-d` options.",
               shell_complete=complete_datasets)
+@click.option('--n-workers', '-j',
```

Comment:
j for jobs ? if so can we match with `--n-jobs` ?

Reply:
I don't know on this one. it is probably cleaner to use `n_jobs` and `j` but I tend to like better the term `workers`.
I leave the choice to you :)

Reply:
I am less familiar with the vocabulary of parallelism but I am used to n_jobs in joblib and sklearn + I like coherence between `-j` and `jobs`, so rather in favor of `n_jobs`

Reply:
```suggestion
@click.option('--n-jobs', '-j',
```